src/components/images/defaults.js

```diff
@@ -0,0 +1,65 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Axes = require('../../plots/cartesian/axes');
+var Lib = require('../../lib');
+var attributes = require('./attributes');
+
+
+module.exports = function supplyLayoutDefaults(layoutIn, layoutOut) {
+
+    if(!layoutIn.images) return;
+
+
+    var containerIn = Array.isArray(layoutIn.images) ?
+            layoutIn.images : [layoutIn.images],
```

Comment:
Ah ok. I figured as a convenience we could accept a single object. 

Strict is good though :+1: